models/location_helpers_test.go

```diff
@@ -307,7 +307,7 @@ func TestBackupLocationValidation(t *testing.T) {
 				Name: "s3-1",
 				BackupLocationConfig: models.BackupLocationConfig{
 					S3Config: &models.S3LocationConfig{
-						Endpoint:   "https://s3.us-west-2.amazonaws.com/mybucket",
```

Comment:
can we add such case to tests? when endpoint contains bucket name and bucket name is passed separately as well

Reply:
the endpoint cannot contain bucket name at all :)